src/components/content.js

```diff
@@ -698,15 +699,25 @@ class Content extends React.Component {
     if (this.props.readOnly) return
     if (!this.isInEditor(event.target)) return
 
-    event.preventDefault()
+    const handleData = (data) => {
```

Comment:
Ah wait, I think that's why you've got the remove/re-add logic for the node. Does that logic work even though those elements are controlled by React?